### DISCUSSION TITLE
When will Dagster support MySQL

### DISCUSSION THREAD

**Discussion by nancydyc** [dagster-io/dagster#3550] [Q&A] [answered]
When will Dagster support MySQL?
#3227

**Comment by nancydyc** [comment]
The work of supporting the MySQL instance in Dagster infrastructure is in progress; yet we don’t have a concrete timeline. We are actively working on adjusting the schema and adding an event listener/poller for MySQL.

**Comment by yuhan** [comment, marked as answer]
MySQL support in Dagster was introduced in [0.11.0](https://docs.dagster.io/changelog#0110).
Check out the API Docs here: https://docs.dagster.io/_apidocs/libraries/dagster-mysql
